Can't build if mousekeys are in constant mode

<!-- Provide a general summary of the bug in the title above. -->

<!--- This template is entirely optional and can be removed, but is here to help both you and us. -->
<!--- Anything on lines wrapped in comments like these will not show up in the final text. -->

## Describe the Bug

Attempting to build the firmware with mousekeys in constant mode gives me this error:
``` $ make preonic/rev3:stephaneleveugle
QMK Firmware 0.7.27
Making preonic/rev3 with keymap stephaneleveugle

arm-none-eabi-gcc.exe (GNU Tools for ARM Embedded Processors 6-2017-q2-update) 6.3.1 20170620 (release) [ARM/embedded-6-branch revision 249437]
Copyright (C) 2016 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Size before:
   text    data     bss     dec     hex filename
      0   49056       0   49056    bfa0 .build/preonic_rev3_stephaneleveugle.hex

Compiling: quantum/audio/muse.c                                                                     [OK]
Compiling: keyboards/preonic/preonic.c                                                              [OK]
Compiling: keyboards/preonic/rev3/rev3.c                                                            [OK]
Compiling: keyboards/preonic/keymaps/stephaneleveugle/keymap.c                                      [OK]
Compiling: quantum/quantum.c                                                                        [OK]
Compiling: quantum/keymap_common.c                                                                  [OK]
Compiling: quantum/keycode_config.c                                                                 [OK]
Compiling: quantum/matrix.c                                                                         [OK]
Compiling: quantum/debounce/sym_g.c                                                                 [OK]
Compiling: quantum/process_keycode/process_audio.c                                                  [OK]
Compiling: quantum/process_keycode/process_clicky.c                                                 [OK]
Compiling: quantum/audio/audio_arm.c                                                                [OK]
Compiling: quantum/audio/voices.c                                                                   [OK]
Compiling: quantum/audio/luts.c                                                                     [OK]
Compiling: quantum/process_keycode/process_music.c                                                  [OK]
Compiling: quantum/encoder.c                                                                        [OK]
Compiling: quantum/process_keycode/process_space_cadet.c                                            [OK]
Compiling: quantum/dip_switch.c                                                                     [OK]
Compiling: tmk_core/common/host.c                                                                   [OK]
Compiling: tmk_core/common/keyboard.c                                                               [OK]
Compiling: tmk_core/common/action.c                                                                 [OK]
Compiling: tmk_core/common/action_tapping.c                                                         [OK]
Compiling: tmk_core/common/action_macro.c                                                           [OK]
Compiling: tmk_core/common/action_layer.c                                                           [OK]
Compiling: tmk_core/common/action_util.c                                                            [OK]
Compiling: tmk_core/common/print.c                                                                  [OK]
Compiling: tmk_core/common/debug.c                                                                  [OK]
Compiling: tmk_core/common/util.c                                                                   [OK]
Compiling: tmk_core/common/eeconfig.c                                                               [OK]
Compiling: tmk_core/common/report.c                                                                 [OK]
Compiling: tmk_core/common/chibios/suspend.c                                                        [OK]
Compiling: tmk_core/common/chibios/timer.c                                                          [OK]
Compiling: tmk_core/common/chibios/bootloader.c                                                     [OK]
Compiling: tmk_core/common/chibios/printf.c                                                         [OK]
Compiling: tmk_core/common/chibios/eeprom_stm32.c                                                   [OK]
Compiling: tmk_core/common/chibios/flash_stm32.c                                                    [OK]
Compiling: tmk_core/common/bootmagic.c                                                              [OK]
Compiling: tmk_core/common/mousekey.c                                                               [OK]
Compiling: tmk_core/common/command.c                                                               tmk_core/common/command.c: In function 'mousekey_console_help':
tmk_core/common/command.c:721:13: error: 'MOUSEKEY_MOVE_DELTA' undeclared (first use in this function)
             MOUSEKEY_MOVE_DELTA, MOUSEKEY_WHEEL_DELTA);
             ^~~~~~~~~~~~~~~~~~~
tmk_core/common/command.c:721:13: note: each undeclared identifier is reported only once for each function it appears in
tmk_core/common/command.c:721:34: error: 'MOUSEKEY_WHEEL_DELTA' undeclared (first use in this function)
             MOUSEKEY_MOVE_DELTA, MOUSEKEY_WHEEL_DELTA);
                                  ^~~~~~~~~~~~~~~~~~~~
tmk_core/common/command.c: In function 'mousekey_console':
tmk_core/common/command.c:764:36: error: 'MOUSEKEY_DELAY' undeclared (first use in this function)
             mk_delay             = MOUSEKEY_DELAY / 10;
                                    ^~~~~~~~~~~~~~
tmk_core/common/command.c:765:36: error: 'MOUSEKEY_INTERVAL' undeclared (first use in this function)
             mk_interval          = MOUSEKEY_INTERVAL;
                                    ^~~~~~~~~~~~~~~~~
tmk_core/common/command.c:766:36: error: 'MOUSEKEY_MAX_SPEED' undeclared (first use in this function)
             mk_max_speed         = MOUSEKEY_MAX_SPEED;
                                    ^~~~~~~~~~~~~~~~~~
tmk_core/common/command.c:767:36: error: 'MOUSEKEY_TIME_TO_MAX' undeclared (first use in this function)
             mk_time_to_max       = MOUSEKEY_TIME_TO_MAX;
                                    ^~~~~~~~~~~~~~~~~~~~
tmk_core/common/command.c:768:36: error: 'MOUSEKEY_WHEEL_MAX_SPEED' undeclared (first use in this function)
             mk_wheel_max_speed   = MOUSEKEY_WHEEL_MAX_SPEED;
                                    ^~~~~~~~~~~~~~~~~~~~~~~~
tmk_core/common/command.c:769:36: error: 'MOUSEKEY_WHEEL_TIME_TO_MAX' undeclared (first use in this function)
             mk_wheel_time_to_max = MOUSEKEY_WHEEL_TIME_TO_MAX;
                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~
 [ERRORS]
 |
 |
 |
make[1]: *** [tmk_core/rules.mk:380: .build/obj_preonic_rev3_stephaneleveugle/common/command.o] Error 1
Make finished with errors
make: *** [Makefile:574: preonic/rev3:stephaneleveugle] Error 1
```
I tried following the doc: https://docs.qmk.fm/#/feature_mouse_keys?id=constant-mode

This is the content of my config.h file:
```
#pragma once
#define MK_3_SPEED
```
<!-- A clear and concise description of what the bug is. -->

## System Information

 - Keyboard:  Preonic
   - Revision (if applicable):  3
 - Operating system: Windows 7
 - AVR GCC version: avr-gcc.exe (AVR_8_bit_GNU_Toolchain_3.6.1_1752) 5.4.0
<!-- Run `avr-gcc --version` to find this out. -->
 - ARM GCC version: arm-none-eabi-gcc.exe (GNU Tools for ARM Embedded Processors 6-2017-q2-update) 6.3.1 20170620 (release) [ARM/embedded-6-branch revision 249437]
<!-- Run `arm-none-eabi-gcc --version` to find this out. -->
 - QMK Firmware version: 0.7.27
<!-- Run `git describe --abbrev=0 --tags` to find this out. -->
 - Any keyboard related software installed? 
   - [ ] AutoHotKey
   - [ ] Karabiner
   - [ ] Other:

## Additional Context

I'm not sure this is a bug or obsolete documentation or me doing something wrong, I just got my preonic.
I also tried adding the missing defines, to no avail.
With this config file content:
```
#pragma once
#define MK_3_SPEED
#define MOUSEKEY_DELAY 300
#define MOUSEKEY_INTERVAL 50
#define MOUSEKEY_MAX_SPEED 10
#define MOUSEKEY_TIME_TO_MAX 20
#define MOUSEKEY_WHEEL_MAX_SPEED 8
#define MOUSEKEY_WHEEL_TIME_TO_MAX 40
#define MOUSEKEY_MOVE_DELTA 5
#define MOUSEKEY_WHEEL_DELTA 1
```
The error became:
```
make preonic/rev3:stephaneleveugle
QMK Firmware 0.7.27
Making preonic/rev3 with keymap stephaneleveugle

arm-none-eabi-gcc.exe (GNU Tools for ARM Embedded Processors 6-2017-q2-update) 6.3.1 20170620 (release) [ARM/embedded-6-branch revision 249437]
Copyright (C) 2016 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Size before:
   text    data     bss     dec     hex filename
      0   49056       0   49056    bfa0 .build/preonic_rev3_stephaneleveugle.hex

Compiling: quantum/audio/muse.c                                                                     [OK]
Compiling: keyboards/preonic/preonic.c                                                              [OK]
Compiling: keyboards/preonic/rev3/rev3.c                                                            [OK]
Compiling: keyboards/preonic/keymaps/stephaneleveugle/keymap.c                                      [OK]
Compiling: quantum/quantum.c                                                                        [OK]
Compiling: quantum/keymap_common.c                                                                  [OK]
Compiling: quantum/keycode_config.c                                                                 [OK]
Compiling: quantum/matrix.c                                                                         [OK]
Compiling: quantum/debounce/sym_g.c                                                                 [OK]
Compiling: quantum/process_keycode/process_audio.c                                                  [OK]
Compiling: quantum/process_keycode/process_clicky.c                                                 [OK]
Compiling: quantum/audio/audio_arm.c                                                                [OK]
Compiling: quantum/audio/voices.c                                                                   [OK]
Compiling: quantum/audio/luts.c                                                                     [OK]
Compiling: quantum/process_keycode/process_music.c                                                  [OK]
Compiling: quantum/encoder.c                                                                        [OK]
Compiling: quantum/process_keycode/process_space_cadet.c                                            [OK]
Compiling: quantum/dip_switch.c                                                                     [OK]
Compiling: tmk_core/common/host.c                                                                   [OK]
Compiling: tmk_core/common/keyboard.c                                                               [OK]
Compiling: tmk_core/common/action.c                                                                 [OK]
Compiling: tmk_core/common/action_tapping.c                                                         [OK]
Compiling: tmk_core/common/action_macro.c                                                           [OK]
Compiling: tmk_core/common/action_layer.c                                                           [OK]
Compiling: tmk_core/common/action_util.c                                                            [OK]
Compiling: tmk_core/common/print.c                                                                  [OK]
Compiling: tmk_core/common/debug.c                                                                  [OK]
Compiling: tmk_core/common/util.c                                                                   [OK]
Compiling: tmk_core/common/eeconfig.c                                                               [OK]
Compiling: tmk_core/common/report.c                                                                 [OK]
Compiling: tmk_core/common/chibios/suspend.c                                                        [OK]
Compiling: tmk_core/common/chibios/timer.c                                                          [OK]
Compiling: tmk_core/common/chibios/bootloader.c                                                     [OK]
Compiling: tmk_core/common/chibios/printf.c                                                         [OK]
Compiling: tmk_core/common/chibios/eeprom_stm32.c                                                   [OK]
Compiling: tmk_core/common/chibios/flash_stm32.c                                                    [OK]
Compiling: tmk_core/common/bootmagic.c                                                              [OK]
Compiling: tmk_core/common/mousekey.c                                                               [OK]
Compiling: tmk_core/common/command.c                                                                [OK]
Compiling: tmk_core/protocol/chibios/usb_main.c                                                     [OK]
Compiling: tmk_core/protocol/chibios/main.c                                                         [OK]
Compiling: tmk_core/protocol/usb_descriptor.c                                                       [OK]
Compiling: tmk_core/protocol/chibios/usb_driver.c                                                   [OK]
Linking: .build/preonic_rev3_stephaneleveugle.elf                                                   [ERRORS]
 |
 | .build/obj_preonic_rev3_stephaneleveugle/common/command.o: In function `mousekey_param_inc':
 | mypath/qmk_firmware/tmk_core/common/command.c:649: undefined reference to `mk_delay'
 | mypath/qmk_firmware/tmk_core/common/command.c:649: undefined reference to `mk_interval'
 | mypath/qmk_firmware/tmk_core/common/command.c:649: undefined reference to `mk_max_speed'
 | mypath/qmk_firmware/tmk_core/common/command.c:649: undefined reference to `mk_time_to_max'
 | mypath/qmk_firmware/tmk_core/common/command.c:649: undefined reference to `mk_wheel_max_speed'
 | mypath/qmk_firmware/tmk_core/common/command.c:649: undefined reference to `mk_wheel_time_to_max'
 | .build/obj_preonic_rev3_stephaneleveugle/common/command.o: In function `mousekey_param_dec':
 | mypath/qmk_firmware/tmk_core/common/command.c:696: undefined reference to `mk_delay'
 | mypath/qmk_firmware/tmk_core/common/command.c:696: undefined reference to `mk_interval'
 | mypath/qmk_firmware/tmk_core/common/command.c:696: undefined reference to `mk_max_speed'
 | mypath/qmk_firmware/tmk_core/common/command.c:696: undefined reference to `mk_time_to_max'
 | mypath/qmk_firmware/tmk_core/common/command.c:696: undefined reference to `mk_wheel_max_speed'
 | mypath/qmk_firmware/tmk_core/common/command.c:696: undefined reference to `mk_wheel_time_to_max'
 | .build/obj_preonic_rev3_stephaneleveugle/common/command.o: In function `mousekey_console':
 | mypath/qmk_firmware/tmk_core/common/command.c:779: undefined reference to `mk_delay'
 | mypath/qmk_firmware/tmk_core/common/command.c:779: undefined reference to `mk_interval'
 | mypath/qmk_firmware/tmk_core/common/command.c:779: undefined reference to `mk_max_speed'
 | mypath/qmk_firmware/tmk_core/common/command.c:779: undefined reference to `mk_time_to_max'
 | mypath/qmk_firmware/tmk_core/common/command.c:779: undefined reference to `mk_wheel_max_speed'
 | mypath/qmk_firmware/tmk_core/common/command.c:779: undefined reference to `mk_wheel_time_to_max'
 | collect2.exe: error: ld returned 1 exit status
 |
make[1]: *** [tmk_core/rules.mk:300: .build/preonic_rev3_stephaneleveugle.elf] Error 1
Make finished with errors
make: *** [Makefile:574: preonic/rev3:stephaneleveugle] Error 1
```
<!-- Add any other relevant information about the problem here. -->

--- a/tmk_core/common/command.c
+++ b/tmk_core/common/command.c
@@ -39,7 +39,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #    include "backlight.h"
 #endif
 
-#ifdef MOUSEKEY_ENABLE
+#if defined(MOUSEKEY_ENABLE) && !defined(MK_3_SPEED)
 #    include "mousekey.h"
 #endif
 
@@ -57,7 +57,7 @@ static void print_version(void);
 static void print_status(void);
 static bool command_console(uint8_t code);
 static void command_console_help(void);
-#ifdef MOUSEKEY_ENABLE
+#if defined(MOUSEKEY_ENABLE) && !defined(MK_3_SPEED)
 static bool mousekey_console(uint8_t code);
 static void mousekey_console_help(void);
 #endif
@@ -78,7 +78,7 @@ bool command_proc(uint8_t code) {
             else
                 return (command_console_extra(code) || command_console(code));
             break;
-#ifdef MOUSEKEY_ENABLE
+#if defined(MOUSEKEY_ENABLE) && !defined(MK_3_SPEED)
         case MOUSEKEY:
             mousekey_console(code);
             break;
@@ -538,7 +538,7 @@ static bool command_console(uint8_t code) {
         case KC_ESC:
             command_state = ONESHOT;
             return false;
-#ifdef MOUSEKEY_ENABLE
+#if defined(MOUSEKEY_ENABLE) && !defined(MK_3_SPEED)
         case KC_M:
             mousekey_console_help();
             print("M> ");
@@ -553,7 +553,7 @@ static bool command_console(uint8_t code) {
     return true;
 }
 
-#ifdef MOUSEKEY_ENABLE
+#if defined(MOUSEKEY_ENABLE) && !defined(MK_3_SPEED)
 /***********************************************************
  * Mousekey console
  ***********************************************************/

